cloud-gateway/src/main/resources/application.yaml

```diff
@@ -0,0 +1,18 @@
+server:
+  port:
+    9191
+
+spring:
+  application:
+    name: CLOUD-GATEWAY
+  cloud:
+    gateway:
+      routes:
```

Comment:
Probably don't need these if we are going to use the gateway as not just a router. Talk with Rado about this.